app/styles/mobile.scss

```diff
@@ -103,11 +103,11 @@
 #footer {
   padding-top: 50px;
   text-align: left;
-  a{
-    color:black;
+  a:hover{
+    color:purple;
   }
   center{
-    color:black;
+    color:white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space